java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6804.html

```diff
@@ -0,0 +1,60 @@
+<p>This rule reports when the Spring <code>@Value</code> annotation is used to inject a simple value that does not contain an expression.</p>
```

Comment:
Let's rewrite this sentence:
This rule reports when the Spring `@Value` annotation injects a simple value that does not contain an expression.